src/table/TableBodyWrapper.jsx

```diff
@@ -9,15 +9,24 @@ import { getCellRenderer } from './cells/renderer';
 import { getBodyStyle } from './styling-utils';
 
 const useStyles = makeStyles({
-  body: (props) => ({
+  tableCell: (props) => ({
     fontSize: props.fontSize,
     color: props.fontColor,
+    '&:hover': {
+      color: props.hoverFontColor,
+    },
+  }),
+  tableRow: (props) => ({
+    '&&:hover': {
+      backgroundColor: props.hoverBackGroundColor,
+    },
   }),
 });
 
 export default function TableBodyWrapper({ tableData, constraints, selectionsAPI, layout, theme }) {
   const { rows, columns } = tableData;
   const classes = useStyles(getBodyStyle(layout, theme));
+  const hoverEffect = layout?.components?.[0]?.hoverEffect;
```

Comment:
```suggestion
  const hoverEffect = layout.components?.[0]?.hoverEffect;
```
layout will always exist, we check that in index before rendering

Reply:
okay, gonna fix

Reply:
hoverEffect does not work. will fix it.